### ISSUE TITLE
Plugin browser download counter

### ISSUE THREAD

**Issue by simondate** [adaptlearning/adapt-cli#86]
The plugin browsers counter only counts downloads through the Adapt-cli tool. It's likely that most users don't use the CLI. Instead getting their plugins through `git clone` or just downloading as a zip. This is problematic as users who see plugins with low download counts may assume they haven't been used at all and may be unstable or unusable.

We're looking for suggestions on how to improve the plugin browser experience. A few ideas options are:

1. Use Github stars
2. Continue to use download to order but hide the download counter
3.  Keep everything the same.

If you have any of your own ideas please add them as a comment below.

**Comment by paul-mediakitchen** [comment]
Be nice to have a unique link to each plugin as currently they all have the same link:

https://www.adaptlearning.org/index.php/plugin-browser/

**Comment by taylortom** [comment]
I don't think any of the GitHub metrics will give us any meaningful sorting -- have you looked at the star count for the core plugins? 

My preferred solution is to hide the download count but continue to order by downloads. Would be nice to introduce additional sorts too, but not sure how easy this is.

**Comment by simondate** [comment]
I agree with Tom. Hiding the counter would still keep the same order (which is probably accurate) but not show incorrect information which can be detrimental to Adapt.
